### PR TITLE
Add favicon

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

**Comment by dmage** [comment]
@bparees this favicon is a simple letter `s` on a red background, let me know if you envision it differently. Once we migrate to 1.16, we can use `embed` for static files.

**Comment by dmage** [comment]
Or we can pick some emoji-favicon, like this https://favicon.io/emoji-favicons/robot/ or https://favicon.io/emoji-favicons/police-car-light/ :)

**Comment by dmage** [comment]
Updated using official logo

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
-#    dnf clean all -y
-RUN mkdir /tmp/src
-COPY . /tmp/src
-RUN cd /tmp/src; make build
-ENTRYPOINT /tmp/src/sippy
-
+WORKDIR /go/src/sippy
+COPY . .
+RUN make build
+ENTRYPOINT ["/go/src/sippy/sippy"]
```
Comment (bparees) [inline Dockerfile:5]: fetchdata.sh and fetchdata-kube.sh will need to be updated for this.  and the deploymentconfig yamls

**Review by bparees — CHANGES_REQUESTED** [review]
thanks! :)

File: Dockerfile (hunk 1 above)
Comment (dmage) [reply Dockerfile:5]: updated

**Pull request merged**